assets/src/styles/network.scss

```diff
@@ -0,0 +1,182 @@
+@import 'common';
+
+$fa-font-path: './fonts' !default;
+@import '../../node_modules/@fortawesome/fontawesome-free/scss/fontawesome';
+@import '../../node_modules/@fortawesome/fontawesome-free/scss/brands';
+@import '../../node_modules/@fortawesome/fontawesome-free/scss/solid';
+@import '../../node_modules/leaflet/dist/leaflet';
+@import '../../node_modules/leaflet.markercluster/dist/MarkerCluster';
+@import '../../node_modules/leaflet.markercluster/dist/MarkerCluster.Default';
+
+@import 'partials/script';
+@import 'partials/slider';
+@import 'partials/social-share';
+@import 'partials/tooltip';
+
+.wdfn-component {
+  &[data-component='network'] {
+    @import './components/network';
+    margin-bottom: 1em;
+  }
+}
+
+/* This fixes an issue with Firefox not hiding long divs. I opened https://github.com/uswds/uswds/issues/2380 .
+ * If it gets addressed this can be removed.
+ */
+.usa-sr-only {
+  overflow: hidden;
+}
+
+body {
+  #main-content {
+    margin-top: 1em;
+    margin-bottom: 1em;
+    padding: 0 1em;
+  }
+
+  .content-container {
+    margin-left: auto;
+    margin-right: auto;
+    @include u-maxw('desktop');
+  }
+
+  #wdfn-alert-banner input:checked ~ * {
```

Comment:
Does the networks page use alerts? Then please remove. If they are, we should consider creating a partials for this so it can be shared.

Reply:
I only see them in the beta release alert, however decided to make a partial just in case we would need them in the future.